proposals/0293-extend-property-wrappers-to-function-and-closure-parameters.md

```diff
@@ -315,11 +315,11 @@ This transformation at the call-site only applies when calling the function dire
 
 #### Passing a projected value argument
 
-Property wrappers can opt into passing a projected-value argument to a property-wrapped parameter.
+Property-wrapper projections are designed to allow property wrappers to provide a representation of the storage type that can be used outside of the context that owns the property-wrapper storage. Typically, projections either expose the backing property wrapper directly, or provide an instance of a separate type that vends more restricted access to the functionality on the property wrapper.
```

Comment:
I think you put it really well on the forums: "a view of the property wrapper from the outside; I just paraphrased it:

Property-wrapper projections ("projections" as in projections of the property wrapper to the context of the wrapped declaration) ..

I just think that with some refinement it would be really insightful for the reader.